evap/staff/templates/staff_semester_view.html

```diff
@@ -11,75 +12,122 @@
-        <a href="{% url "evap.staff.views.semester_delete" semester.id %}" class="btn btn-sm btn-danger" {% if not semester.can_staff_delete %} disabled="disabled" {% endif %}>{% trans "Delete Semester" %}</a>
-        <a href="{% url "evap.staff.views.semester_archive" semester.id %}" class="btn btn-sm btn-danger" {% if not semester.is_archiveable %} disabled="disabled" {% endif %}>{% trans "Archive Semester" %}</a>
-        &nbsp;
-        <a href="{% url "evap.staff.views.course_create" semester.id %}" class="btn btn-sm btn-default" {{ disable_if_archived }}>{% trans "Create Course" %}</a>
-        <a href="{% url "evap.staff.views.semester_import" semester.id %}" class="btn btn-sm btn-default" {{ disable_if_archived }}>{% trans "Import Courses" %}</a>
-        <a href="{% url "evap.staff.views.semester_assign_questionnaires" semester.id %}" class="btn btn-sm btn-default" {{ disable_if_archived }}>{% trans "Assign Questionnaires" %}</a>
-        &nbsp;
-        <a href="{% url "evap.staff.views.semester_lottery" semester.id %}" class="btn btn-sm btn-default">{% trans "Lottery" %}</a>
-        &nbsp;
-        <a href="{% url "evap.staff.views.semester_todo" semester.id %}" class="btn btn-sm btn-default" {{ disable_if_archived }}>{% trans "Todo" %}</a>
-        &nbsp;
-
-        <div class="btn-group">
-            <a href="{% url "evap.rewards.views.semester_reward_points" semester.id %}" class="btn btn-sm btn-default">{% trans "Reward Points" %}</a>
-            <div class="btn-group btn-toggle">
-                <a href="{% url "evap.rewards.views.semester_activation" semester.id "on" %}" class="btn btn-sm btn-default {% if rewards_active %}active{% endif %}">{% trans "On" %}</a>
-                <a href="{% url "evap.rewards.views.semester_activation" semester.id "off" %}" class="btn btn-sm btn-default {% if rewards_active|is_false %}active{% endif %}">{% trans "Off" %}</a>
-            </div>
+    {% if num_courses > 0 %}
+    <div class="panel panel-info">
+        <div class="panel-heading">
+            <h3 class="panel-title">{% trans "Overview" %}</h3>
         </div>
-    </div>
-
-    {% if courses_by_state %}
-        <ul class="nav nav-tabs">
-            {% for state, courses in courses_by_state %}
-                <li><a href="#tab{{ forloop.counter }}" data-tab="{{ forloop.counter }}" data-toggle="tab">{{ state|statename }} ({{ courses|length }})</a></li>
-            {% endfor %}
-        </ul>
-
-            <div class="tab-content">
-            {% for state, courses in courses_by_state %}
-                <div id="tab{{ forloop.counter }}" class="tab-pane">
-                <table class="table table-striped">
+        <div class="panel-body">
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th class="col-sm-3">{% trans "Evaluation periods" %}</th>
```

Comment:
"the evaluation periods of the courses are from ... to ...". but of course one could also say "the evaluation period of the semester is from ... to ...".
i'll change it :)